client/src/components/ProjectWizard/TdmCalculationWizard.js

```diff
@@ -79,10 +86,40 @@ const useStyles = createUseStyles({
   },
   buttonContainer: {
     marginTop: "5px"
+  },
+  lastSaved: {
+    fontSize: "14px",
+    color: "#6F6C64"
+  },
+  lastSavedContainer: {
+    margin: "0 auto"
+  },
+  navArrow: {
+    marginTop: "6px"
   }
 });
 
 const TdmCalculationWizard = props => {
+  const [dateModified, setDateModified] = useState("placeholder");
```

Comment:
Just use an empty string next time. Otherwise this will happen: 
![image](https://user-images.githubusercontent.com/45305029/85373646-6c438c80-b4e8-11ea-8b51-4af096e26e77.png)